pkg/chroot/root.go

```diff
@@ -0,0 +1,18 @@
+package chroot
+
+import (
+	"errors"
+	"os"
+)
+
+func TmpDirInHome() (string, error) {
```

Comment:
We may want [`os.UserHomeDir()`](https://pkg.go.dev/os#UserHomeDir) instead, which uses `$HOME` and other heuristics.